examples/tests/cloud-push-pull/azure-container-registry.earth

```diff
@@ -0,0 +1,12 @@
+FROM earthly/dind:alpine-main
```

Comment:
I would use the released version of the dind image: `earthly/dind:alpine`.

Reply:
Good catch